h/formatters/annotation_moderation.py

```diff
@@ -35,7 +36,9 @@ def preload(self, ids):
         return flag_counts
 
     def format(self, annotation_resource):
-        if not self._has_permission("moderate", annotation_resource.group):
+        if not self._has_permission(
+            Permission.Annotation.MODERATE, annotation_resource
+        ):
```

Comment:
Wait, so was this totally broken before?

Reply:
No, it works, but just works weird.

The `Permission.Annotation.MODERATE` is actually 100% derived from the presence of the `Permission.Group.MODERATE` on the relevant group. You can see it's copied over here:

https://github.com/hypothesis/h/blob/master/h/traversal/annotation.py#L71

Before this was nearly impossible to spot, as both we're just "moderate". So the two things are actually 100% aligned (at the moment), but we should be checking with the annotation context not the group model object. Or we should never have bothered making a distinction between the two.